docs/ADOPTING_DARIA.md

```diff
@@ -0,0 +1,43 @@
+# Adopting DARIA
+
+If you are a fund considering adopting DARIA as your case management system of record, this may answer common questions about how things work. We also have a sandbox/training app where no actual patient data lives for experimenting. [FILL IN ANY NECESSARY DETAIL]
```

Comment:
Do we want to get into any detail about who to contact? We've created an email account in the ddf domain to be a catchall for DARIA implementation things. If you think how to move forward details and pace to expect would be helpful, then I can comment with some verbiage :)